sources/views/contacts.js

```diff
@@ -25,21 +25,19 @@ export default class Contacts extends JetView {
 								);
 							},
 							css: "webix_shadow_medium app_start",
-							click: function (id) {
-								this.$scope.setParam("id", id, true);
+							click: (id) => {
```

Comment:
you either need this or select event. not both. let's settle on the event and I still recommend onAfterSelect since unselecting after remove() does not work